content/docs/authorization/cli.md

```diff
@@ -615,7 +617,9 @@ karavictl storage [flags]
 ##### Options
 
 ```
-  -h, --help   help for storage
+  --addr string   address of the csm-authorization storage service
```

Comment:
Now that addr is the same for tenant and storage service, lets keep the description consistent to be `Address of the server (default "localhost")`

Reply:
fixed